apps/evm/src/containers/Layout/Sidebar/NavLink/index.tsx

```diff
@@ -22,15 +22,10 @@ export const NavLink: React.FC<NavLinkProps> = ({
 
   return (
     <RRNavLink
-      className={({ isActive }) =>
-        cn(
-          'hover:bg-lightGrey active:bg-lightGrey flex items-center justify-center whitespace-nowrap px-6 py-4 h-14 w-14 rounded-xl p-0 xl:relative xl:h-auto xl:w-full xl:rounded-none xl:px-8 xl:py-4 xl:font-semibold',
-          isActive
-            ? 'bg-lightGrey text-offWhite xl:before:bg-blue xl:before:absolute xl:before:bottom-0 xl:before:left-0 xl:before:top-0 xl:before:w-1 xl:before:rounded-br-lg xl:before:rounded-tr-lg'
-            : 'text-grey',
-          className,
-        )
-      }
+      className={cn(
+        'hover:bg-lightGrey active:bg-lightGrey flex items-center justify-center text-grey whitespace-nowrap px-6 py-4 h-14 w-14 rounded-xl p-0 xl:relative xl:h-auto xl:w-full xl:rounded-none xl:px-8 xl:py-4 xl:font-semibold',
+        className,
+      )}
```

Comment:
Feels weird to lose the active styling on all items because of the new item 🤔

Reply:
I agree yes. I created this PR to get a test environment for this.
We're discussing the matter further in the design channel :)

Reply:
After speaking with Brad and Pavel, the choice has been made to keep the highlighting feature and remove the "Staked ETH pool" instead.
I've also created a separate Lido Market page to prevent clashing with the Isolated Pools page.